Default (Linux).sublime-keymap

```diff
@@ -1,5 +1,10 @@
 [
     {
+        "keys": ["ctrl+shift+o"],

```

Comment:
I'm generally against default keymaps that aren't limited by context.  I personally bind FuzzyFileNav as you have done here, but I don't force that on anyone.

Reply:
I can follow your arguments. Key bindings may not be too important if the commands are found in the command pallet.

Reply:
Yeah, people can bind whatever they want.